src/components/profile/index.js

```diff
@@ -83,16 +108,18 @@ export default class SignUp extends React.Component {
   }
 
   handleSignUp = async () => {
-    const { signUp, history } = this.props
+    const { signUp, history }: ProfilePropsType = this.props
     const { email, password } = this.state
 
+    // $FlowFixMe Cannot call signUp because undefined [1] is not a function.
     const isSigned = await signUp({ email, password })
     if (isSigned) {
+      // $FlowFixMe Cannot call history.push because property push is missing in undefined [1].
```

Comment:
Same. History props is optional. Make it required by removing `?` or check that it`s not undefined before making this call

Reply:
signUp is removed because this component does not have form functionality